script/gen_requirements_all.py

```diff
@@ -307,7 +322,7 @@ def generate_requirements_list(reqs):
     return "".join(output)
 
 
-def requirements_output(reqs):
+def requirements_output() -> str:
```

Comment:
Removed the argument, as it is unused.

Reply:
That doesn't sound like something that belongs in a type-hinting PR 😁 

I think it's there for symmetry with the other generators.

Reply:
I chose to include it, as I had to change it anyway, and didn't see the point of adding type hints to an usused argument.

Reply:
I think it's fine. Thanks @epenet